README.md

```diff
@@ -13,13 +13,15 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 - [Installation](#installation)
   - [Prerequisites](#prerequisites)
   - [Installing Minishift](#installing-minishift)
-    - [Installing via Homebrew](#installing-via-homebrew)
+    - [Installing with Homebrew](#installing-with-homebrew)
 - [Quickstart](#quickstart)
   - [Starting Minishift](#starting-minishift)
   - [Deploying a sample application](#deploying-a-sample-application)
   - [Reusing the Docker daemon](#reusing-the-docker-daemon)
 - [Documentation](#documentation)
 - [Known Issues](#known-issues)
+  - [Bugs](#bugs)
+  - [Unsupported features](#unsupported-features)
```

Comment:
This section was already in the README, I just separated it out and gave it a title.